subgraph/packages/subgraph-all-seasons/subgraph.yaml

```diff
@@ -4,6 +4,41 @@ repository: https://github.com/hoprnet/hopr-devrel
 schema:
   file: ./schema.graphql
 dataSources:
+  - kind: ethereum/contract
+    name: HoprStakeSeason8
+    network: gnosis
+    source:
+      abi: HoprStakeBase
+      address: "0xdc8f03f19986859362d15c3d5ed74f26518870b9"
+      startBlock: 28930303
```

Comment:
Use the block number where the contract is deployed. It's `28909156` for s8 contract , see https://gnosisscan.io/tx/0xd31978dbc394d239870b026edffd39cdd582ef2d119a58aaca83ba481276f899